### ISSUE TITLE
Detect package.json configuration?

### ISSUE THREAD

**Issue by niftylettuce** [prettier/vim-prettier#41]
Can this detect configuration for prettier in `package.json`?

```js
  // ...
  "xo": {
    "extends": "prettier",
    "plugins": ["babel", "prettier"],
    "rules": {
      "prettier/prettier": [ // <---------- HERE
        "error",
        {
          "singleQuote": true,
          "bracketSpacing": true,
          "trailingComma": "none"
        }
      ],
      "max-len": ["error", {"code": 80, "ignoreUrls": true}]
    },
```

**Comment by mitermayer** [comment]
Hi @niftylettuce , 

The currently released version of prettier '1.5.3' does not yet support this functionality. 

Once the next prettier version is released (https://github.com/prettier/prettier) it will include support for custom configurations (https://github.com/prettier/prettier/issues/2589) since that functionality was integrated on https://github.com/prettier/prettier/pull/2434. 

As soon as thew new prettier version is released I will then start working on enabling custom configurations by leveraging from that API.

Thank you for submitting this issue, will close this for now since it is not currently applicable but feel free to still comment on it.

**Comment by mitermayer** [comment]
Now with new prettier version 1.6 been released we now will be able to have some level of custom configurations. 

more information can be found on https://github.com/prettier/vim-prettier/issues/51
